packages/@expo/cli/src/api/rest/__tests__/client-test.ts

```diff
@@ -58,70 +52,65 @@ it('converts Expo APIv2 error to ApiV2Error (invalid password)', async () => {
   expect.assertions(3);
 
   try {
-    await fetchAsync('test', {
-      method: 'POST',
-    });
+    await fetchAsync('test', { method: 'POST' });
   } catch (error: any) {
-    assert(error instanceof ApiV2Error);
-
+    expect(error).toBeInstanceOf(ApiV2Error);
     expect(error.message).toEqual('Your username, email, or password was incorrect.');
     expect(error.expoApiV2ErrorCode).toEqual('AUTHENTICATION_ERROR');
   }
-  expect(scope.isDone()).toBe(true);
 });
 
 it('does not convert non-APIv2 error to ApiV2Error', async () => {
-  const scope = nock(getExpoApiBaseUrl()).post('/v2/test').reply(500, 'Something went wrong');
+  nock(getExpoApiBaseUrl()).post('/v2/test').reply(500, 'Something went wrong');
 
-  expect.assertions(1);
+  expect.assertions(2);
 
   try {
-    await fetchAsync('test', {
-      method: 'POST',
-    });
+    console.log(await fetchAsync('test', { method: 'POST' }));
   } catch (error: any) {
-    expect(error).toBeInstanceOf(FetchError);
+    expect(error).toBeInstanceOf(Response);
```

Comment:
@EvanBacon `node-fetch` seems to throw the response for some types of statuses. I've added that behavior back for just the rest API client. But, maybe we should wrap it in a proper error?